tests/conftest.py

```diff
@@ -4,7 +4,7 @@
 from packaging import version
 
 
-DEFAULT_NETBOX_VERSIONS = "2.7, 2.8, 2.9, 2.10"
+DEFAULT_NETBOX_VERSIONS = "2.10, 3.0, 3.1"
```

Comment:
Are these the only versions we want to test against? We could leave more in here and with the github matrix strategy it wouldn't really slow down the test suite runs on github, only the local runs if someone was to run the whole suite.

Reply:
What happened to 2.11? But otherwise, I'm cool with sticking to latest three major releases. Probably don't want to be unnecessarily rough on free resources. :)